.github/workflows/canary-integration-test.yml

```diff
@@ -752,6 +752,44 @@ jobs:
       - name: check-ownerreferences
         run: tests/scripts/github-action-helper.sh check_ownerreferences
 
+      - name: teardown cluster with cleanup policy
+        run: |
+          kubectl -n rook-ceph patch cephcluster rook-ceph --type merge -p '{"spec":{"cleanupPolicy":{"confirmation":"yes-really-destroy-data"}}}'
+          kubectl -n rook-ceph delete cephcluster rook-ceph
+          kubectl -n rook-ceph logs deploy/rook-ceph-operator
+          tests/scripts/github-action-helper.sh wait_for_cleanup_pod
+          kubectl -n rook-ceph delete --ignore-not-found=true -f deploy/examples/operator.yaml
```

Comment:
To help reduce the duplication across the tests, could you refactor at least some of this into `tests/scripts/github-action-helper.sh`?